src/atoms/Photography/photography.module.css

```diff
@@ -0,0 +1,8 @@
+.photography {
+  text-align: center;
+  background-image: url("../../images/image_1.jpg");
+  background-repeat: no-repeat;
+  background-size: 776px 214px;
```

Comment:
These values should be migrated to custom properties in this same file.

Reply:
Do you mean pass these values by component props?

Reply:
Done.